etc/crow.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for crow
+# This file is overwritten after every install/update
+# Persistent local customizations
+include crow.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/crow
```

Comment:
can you add a corresponding blacklist to disable-programs.inc or another place you deem appropriate?

Reply:
i have removed them as they are not blacklisted, thanks